src/registry/map/edit.tsx

```diff
@@ -174,4 +179,36 @@ const UseConfigDefaultMapSettings: React.FC<{}> = () => {
   );
 };
 
+const TileLayer: React.FC = () => {
+  const {getMapTileLayers} = useContext(BuilderContext);
+  const {value: options, loading, error} = useAsync(async () => await getMapTileLayers(), []);
+  if (error) {
+    throw error;
+  }
+
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'tileLayerIdentifier' builder field",
+    defaultMessage: 'The tile layer to use as the map.',
```

Comment:
```suggestion
    defaultMessage: 'The tile layer is responsible for showing the map parts at particular coordinates and zoom levels.',
```

Reply:
Mhh, yeah i agree that the defaultMessage is pretty empty right now.. (and should be changed).
But i'm not sure if we should mention the coordinates and zoom levels. It isn't really connected to each other..

Maybe something like `The tile layer determines the used map background.` is more fitting